modin/pandas/general.py

```diff
@@ -370,12 +370,32 @@ def unique(values):  # noqa: PR01, RT01, D200
     return Series(values).unique()
 
 
-@_inherit_docstrings(pandas.value_counts, apilink="pandas.value_counts")
+# adding docstring since pandas docs doesn't have web section for this function
```

Comment:
Pottentially it can be done, but it will confuse the user since `pandas.value_counts` and `pandas.Series.value_counts` APIs are slightly different (for example `pandas.value_counts` uses additional `values` parameter), but this link to pandas is aimed to show the user API description.

Reply:
Ok, got it, thanks